src/generate-common-seo.js

```diff
@@ -53,5 +53,10 @@ export function generateStructuredData(config) {
       sameAs: socialLinks ? Object.values(socialLinks) : []
     },
     enableNewsArticle: !!themeConfig['structured_data_news_article'],
+    website: {
+      url: config["sketches-host"],
```

Comment:
Ideally this is not the sketches host, but the frontend host

Reply:
This would be the current site url, would it?

Reply:
All malibu based apps currently have the sketches host set to the publisher website. We don't use the `quintype.io` domain explicitly.  And `organization` was already using sketches_host for its url value.  If that schema is working fine, then this should also be ok. Lemme know if BQ/TQ has a different setup.

Reply:
Yup, the sketches host is set to the publisher website as Amith said and it's working fine for `organisation` and as far as i have checked, working fine for `website` too.